src/utils/translations.ts

```diff
@@ -1,10 +1,15 @@
 import type { Lang } from "./languages"
 import type { Direction } from "../types"
 
+import i18nConfigs from "../../i18n/config.json"
+
 export type TranslationKey = string
 
 export const isLangRightToLeft = (lang: Lang): boolean => {
-  return lang === "ar" || lang === "fa"
+  const langConfig = i18nConfigs.filter(
+    (language) => language.hrefLang === lang
+  )
```

Comment:
We may also want to guard (or throw an error) against the case where our filtered array is empty (no matches)

Maybe:

```ts
if (!langConfig.length) throw new Error("Language code not found")
```

or

```ts
if (!langConfig.length) return "rtl"
```

Reply:
I think ill do the first one with throwing an error. Think this would make it easier to debug as this would be an issue. Thanks for the suggestion